use_cases/gretel.json

```diff
@@ -2,8 +2,8 @@
   "cards": [
     {
       "gtmId": "use-case-synthetic",
-      "title": "Generate synthetic data",
-      "description": "Create highly accurate synthetic training data for ML models.",
+      "title": "Generate synthetic tabular data",
+      "description": "Create highly accurate synthetic tabular training data for ML models.",
       "cardType": "Console",
       "imageName": "synthetics.png",
       "icon": "synthetics.png",
```

Comment:
I think we are missing the new buttons on this one

Reply:
pulled in the other previous commit, should be good now!